tests/integ/test_tf_script_mode.py

```diff
@@ -22,9 +23,8 @@
 from sagemaker.tensorflow import TensorFlow
 from six.moves.urllib.parse import urlparse
 from sagemaker.utils import unique_name_from_base
-import tests.integ as integ
-from tests.integ import kms_utils
-import tests.integ.timeout as timeout
+
+import tests.integ
```

Comment:
Nice. Another way I like to do as well is:
```python
from test import integ
```

Reply:
that works too. i'm long winded so i generally prefer fully FQ names.